README.md

```diff
@@ -1,2 +1,138 @@
-# WildVista
- Explore Famous US National Parks and Activities
+<a name="readme-top"></a>
+<div align="center">
+  <br/>
+
+  <h3><b>Capstone Project: Wild Vista</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone Project: Wild Vista <a name="about-project"></a>
+
+The Wild Vista Explorer is a web application that allows users to explore and discover national parks in the United States. It provides a user-friendly interface for browsing parks, viewing park details, and searching for parks based on various criteria.
+
+[Check this quick video about the features](https://www.loom.com/share/3374d04999774cdb9ddba56f64e3fca9)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- React: JavaScript library for building user interfaces.
+- Redux: State management library for managing application state.
+- React Router: Library for routing and navigation in a React application.
+- Axios: Promise-based HTTP client for making API requests.
+- Leaflet: JavaScript library for interactive maps.
+- Jest and React Testing Library: Testing frameworks for unit and integration testing.
+- CSS Modules: CSS styling approach that scopes styles to individual components.
+- Responsive Design: CSS media queries and responsive layout techniques for optimal viewing across device
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Park Listings: Browse a list of national parks withing all the states of the U.S.
+- Park Details: View detailed information about each park, including descriptions, total activities and topics.
+- Search Functionality: Search for parks based on the state code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://wild-vista.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To begin, clone the repository and install all dependencies by running `npm install`. Once the installation is complete, run the project using `npm start`.
+
```

Comment:
- [OPTIONAL] I think it would be better if you explain more about how to clone the repository and how users can get access to it locally, that will help even people who have difficulty of cloning or other things 🙃